client/src/components/Nav/SearchBar.jsx

```diff
@@ -37,8 +48,15 @@ const SearchBar = forwardRef((props, ref) => {
         placeholder="Search messages"
         onKeyUp={handleKeyUp}
       />
+      <X
+        className={`h-5 w-5 absolute right-3 cursor-pointer ${showClearIcon ? 'opacity-100' : 'opacity-0'} transition-opacity duration-1000`}
+        onClick={() => {
+          setSearchQuery('');
+          clearSearch();
```

Comment:
Do these lines do the same thing? also I'm not sure I can see a clearSearch() function included on this script anywhere?

Reply:
I don't think so, dannya used it in ```handleKeyUp``` so I thought that was the best way to clear the input

Reply:
ah I see the function included above now, just wanted to check it was linking somewhere